docker-machine-nfs.sh

```diff
@@ -300,6 +310,17 @@ lookupMandatoryProperties ()
     return
   fi
 
+  if [ "$prop_machine_driver" = "vmwarevsphere" ]; then
+    prop_nfshost_ip=${prop_use_ip:-}
```

Comment:
The fact you pass the ip, is it because this suits your current needs, or because it's simply not possible to figure out the ip?

Reply:
I could not think of a way to reliably determine which ip address a user would want to publish with. What if a user is using a VPN and has multiple networks that they are connected to? How would we determine which one to use?

Reply:
Clear. Thanks.